### DISCUSSION TITLE
hostname is not changing

### DISCUSSION THREAD

**Discussion by stephanlamoureux** [Gobidev/pfetch-rs#58] [Q&A] [answered]
I'm using Arch/Hyprland and I added the env to my .zshrc:

`export HOSTNAME="ThinkPad X1 Carbon Gen 9"`

but when I use pfetch it still displays as:
`host   20XW003HUS ThinkPad X1 Carbon Gen 9`

I also tried adding it to my hyprland.conf as `env = HOSTNAME, ThinkPad X1 Carbon Gen 9`

`echo $HOSTNAME`  does return the correct string.

**Comment by Gobidev** [comment, marked as answer]
I agree that this is a bit confusing.

`host` in pfetch does not display your hostname, the hostname is only part of the `title`, like in `user@system`, where `system` can be changed with the `HOSTNAME` variable. The `host` field displays the name of your hardware system as reported by linux and can currently not be manually overwritten.

If you really need to change the `host` manually, you could compile the project yourself with a fixed host or open an issue for an option to set the `host` manually.

**Comment by Un1q32** [comment]
In my fork of the original pfetch I have a variable PF_HOST to control the host text.  I might make a patch to pfetch-rs to add this functionality eventually.
